src/features/client/components/RestaurantInspectForLanding/index.jsx

```diff
@@ -0,0 +1,47 @@
+import { Box, Flex, Heading } from '@chakra-ui/react';
```

Comment:
Not needed at all, when clicked a discovery item, the user should be redirected to restaurant and added dish to basket (basket logic not finalized atm, when it will be implement, create a ticket for future for now). TLDR remove component, introduce navigation on click of discovery items